easybuild/easyconfigs/n/NAMD/NAMD-2.12-intel-2017a-nompi.eb

```diff
@@ -0,0 +1,29 @@
+name = 'NAMD'
+version = '2.12'
+versionsuffix = '-nompi'
```

Comment:
openmp is not enabled since charm multicore doesn't support it, if i remember correctly right now.

suffix changed, usmpi: False explicit